test/execute.js

```diff
@@ -441,7 +441,7 @@ describe('execute', () => {
       })
     })
 
-    it('should handle requestContentType', function () {
+    it('should not add content-type with no form-data or body param', function () {
```

Comment:
ok... but this test includes a `requestContentType`. If that is present, it should always set the header.